### PR TITLE
Implement the glow around focused colors in highlight popup

### PR DESCRIPTION
**Pull request by PiotrKozlowski**
for: https://github.com/openstax/unified/issues/1494

### CODE REVIEW

File: src/app/content/highlights/components/ColorIndicator.tsx [hunk 1]
```diff
@@ -20,21 +20,44 @@ const CheckIcon = styled(Check)`
   display: none;
 `;
 
+interface FocusedStyleProps extends Pick<Props, 'style' | 'size' | 'shape'> {}
```
Comment (jarosik10) [inline src/app/content/highlights/components/ColorIndicator.tsx:23]: Making
```
interface FocusedStyleProps {
  style: typeof highlightStyles[number];
  size?: 'small'
  shape?: 'square' | 'circle';
}
```
and then `Props` to extend it feels more clear, but I think it's debatable and it doesn't really matter. :P

Comment (jarosik10) [reply src/app/content/highlights/components/ColorIndicator.tsx:23]: Btw. should `size` and `shape` be optional?

Comment (PiotrKozlowski) [reply src/app/content/highlights/components/ColorIndicator.tsx]: I've did what you've suggested but I've changed the name of the interface to `StyleProps` so it makes more sense he they are separated from the `Props`

Comment (PiotrKozlowski) [reply src/app/content/highlights/components/ColorIndicator.tsx]: > Btw. should size and shape be optional?

There is support for `undefined` values and this PR is not really doing anything about these values so I think yes, they should stay optional

**Review by jarosik10 — APPROVED** [review]

**Pull request merged**